### ISSUE TITLE
Does this plugin support multi version of Scala?

### ISSUE THREAD

**Issue by He-Pin** [davidB/scala-maven-plugin#779]
Motivation:
As SBT, do a release with `scala 2.13.x` and `scala 3.3.x`.

**Comment by slandelle** [comment]
No it doesn't. Contributions welcome.

**Comment by slandelle** [comment]
Closing for now. Will reopen if we ever get a contribution.
But IMHO, this plugin is meant to bring standard Scala to maven users and you can't expect it to cover all the possible feature of sbt. If you want to do advanced Scala such as cross-compilation, Scala.js, Scala Native, and such, you should consider moving to sbt (or another Scala build tool).

**Comment by jozic** [comment]
This can be achieved by having a separate profile for 2.12 and invoking deploy/install with the profile enabled.

**Comment by slandelle** [comment]
Indeed. But then I'm not sure if we currently support having multiple source directories, eg `shared`, `2.12`, `2.13`, `3`.
